layouts/index.html

```diff
@@ -1,12 +1,5 @@
-{{define "main"}}
-<div class="row">
-{{ partial "tag-cloud.html" . }}
-</div>
-<div class="row">
-    {{ range .Site.RegularPages.ByLastmod.Reverse }}
-    {{ .Render "thumbnail"}}
-    {{ end }}
-</div>
-
-{{ partial "contribute.html" . }}
-{{ end }}
+{{ define "main" }}
+  {{ $section_to_display := first 1000 .Site.RegularPages.ByLastmod.Reverse }}
+  {{ $.Scratch.Add "section_to_display" $section_to_display }}
```

Comment:
Good call, and just rushing on my part for the easiest way to solve the problem when I conceived of this pattern. I'll need to fix this in the gohugo theme, too.